update-helper.js

```diff
@@ -3,9 +3,14 @@ var execSync = require('child_process').execSync;
 var io = require('socket.io-client');
 var inquirer = require('inquirer');
 
+var args = [];

```

Comment:
I'm sorry for this mutable top-scope var, but I found it hard to navigate around the already existing mutable code which forceUpdate() implements